frontend/src/components/Overlay/OverlayHeader.css

```diff
@@ -39,3 +41,36 @@
 	width: 1.25rem;
 	height: 1.25rem;
 }
+
+@media (width < 93.75rem) {
+	.overlay-header {
+		height: 4rem;
+	}
+}
+
+@media (width < 87.5rem) {
+	.overlay-header {
+		height: 3rem;
+	}
+}
+
+@media (width < 62.5rem) {
+	.overlay-header {
+		height: auto;
+		padding: 0 0.125rem;
+	}
+
+	.overlay-header h1 {
+		border-top-left-radius: 0.625rem;
```

Comment:
Hmm, interesting. Is the header overflowing the header boundaries without this? Feels a bit strange to need a single corner radius which needs to match some other radius, instead of using e.g. overflow: hidden.

Reply:
ah, yep, that's not needed, that was from development, if you give the header a background colour you see if overflowing, but that's obviously not an issue if the header does not have a background color. Will remove it.   
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/1db6b02c-c685-4884-9b39-5317195f5a14)